dev/Pager/PrototypePager/PrototypePager.Events.cs

```diff
@@ -0,0 +1,178 @@
+﻿using Microsoft.UI.Xaml.Controls;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Diagnostics;
+using System.Text;
+using Windows.UI;
+using Windows.UI.Notifications;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Automation.Peers;
+using Windows.UI.Xaml.Automation.Provider;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Media;
+
+namespace MUXControlsTestApp
+{
+    public sealed partial class PrototypePager : Control
+    { 
+        private static void OnSelectedIndexChanged(DependencyObject sender, DependencyPropertyChangedEventArgs args)
+        {
+            (sender as PrototypePager).PreviousPageIndex = (int)args.OldValue - 1;
+        }
+        private void OnPagerDisplayModeChanged()
+        {
+            switch (this.PagerDisplayMode)
```

Comment:
This looks like duplicated code from PrototypePager.cs ? can we reuse the method instead ?

Reply:
Done!